src/scribe_data/cli/main.py

```diff
@@ -0,0 +1,61 @@
+"""
+Setup and commands for the Scribe-Data command line interface.
+
+.. raw:: html
```

Comment:
This should copyright notice is the one I was talking about. We should have it at the top of each Python file :)